diff --git a/backend/modularity/system/JNPF.Systems/Common/FileService.cs b/backend/modularity/system/JNPF.Systems/Common/FileService.cs
index 87b6d079..e002eb28 100644
--- a/backend/modularity/system/JNPF.Systems/Common/FileService.cs
+++ b/backend/modularity/system/JNPF.Systems/Common/FileService.cs
@@ -445,7 +445,7 @@ public class FileService : IFileService, IDynamicApiController, ITransient
     [NonAction]
     public async Task UploadFileByType(string uploadFilePath, string directoryPath, string fileName)
     {
-        FileStream? file = new FileStream(uploadFilePath, FileMode.Open, FileAccess.Read, FileShare.Read);
+        using var file = new FileStream(uploadFilePath, FileMode.Open, FileAccess.Read, FileShare.Read);
         await _fileManager.UploadFileByType(file, directoryPath, fileName);
     }
     #endregion
